src/manifolds/Stiefel.jl

```diff
@@ -162,6 +162,56 @@ function exp!(::Stiefel{n,k}, q, p, X) where {n,k}
     )
 end
 
+@doc raw"""
+    get_basis(M::Stiefel{n,k,ℝ}, p, B::DefaultBasis) where {n,k}
+
+Create the default basis using the parametrization for any $X ∈ T_p\mathcal M$.
+Set $p_\bot \in ℝ^{n\times(n-k)}$ the matrix such that the $n\times n$ matrix of the common
+columns $[p\ p_\bot]$ is an ONB.
```

Comment:
Can this be a `DefaultOrthonormalBasis` then?

Reply:
No. the [p\ p_\bot] is an ONB of R^n (that is the characteristic of p anyways and the goal of constructing p\bot) but all that is just a helper to construct the tangent basis and has nothing to do with the resulting basis.

In fact the resulting basis is neither orthogonal nor normalised.

Reply:
I just noticed, that `a` and `b` are indeed orthonormal coordinates, so yes it is an ONB, I will adapt that.